edge/pkg/metamanager/client/serviceaccount.go

```diff
@@ -49,7 +49,7 @@ func (c *serviceAccountToken) DeleteServiceAccountToken(podUID types.UID) {
 	for _, sa := range *svcAccounts {
 		var tr authenticationv1.TokenRequest
 		err = json.Unmarshal([]byte(sa.Value), &tr)
-		if err != nil {
+		if err != nil || tr.Spec.BoundObjectRef != nil {
```

Comment:
```suggestion
		if err != nil || tr.Spec.BoundObjectRef == nil {
```
Seems it should be nil?

Reply:
Is it also need to fix for master?

Reply:
> Seems it should be nil?

fixed. 😢

Reply:
> Is it also need to fix for master?

PR #5133 fixs it for master.